proto/process/agent.proto

```diff
@@ -29,14 +29,9 @@ message ResCollector {
 	CollectorStatus status = 3;
 }
 
-// CollectorProcHint is an enum that gives "hints" to backend services on how to handle
-// the parent message.
-enum CollectorProcHint {
-        hintUnspecified = 0;
-        hintProcessDiscovery = 1;
-}
-
 message CollectorProc {
+        reserved 13;
```

Comment:
Mind adding a comment like the ones for fields 8/9 like
```	
reserved 8, 9; // Old, optional metadata fields
```

Reply:
Done